classes/product.js

```diff
@@ -0,0 +1,41 @@
+class Product
+{
+    /**
+     * @constructor Represents a product.
+     *
+     * @param {string} name - The display name for this product.
+     * @param {int} quantity - The amount of this product
+     * @param {int} perishable - The number of days a perishable good remains fresh. 0 for infinite.
+     */
+    constructor(name, quantity, perishable = 0)
```

Comment:
Maybe have like generic measurement in 'units'.
For example: a container has a capacity of 100 units. A bike equals 5 units and a chair equals 2 units, then one container can contain:

- 20 bikes or
- 16 bikes and 10 chairs

Has the advantage that we don't have to deal with length, width and height of products for now.